components/icons/discord-social.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <svg
+    width="14"
+    height="16"
+    viewBox="0 0 14 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M5.39125 6.85715C5.8484 6.85715 6.21865 7.2 6.2103 7.61905C6.2103 8.0381 5.84915 8.38095 5.39125 8.38095C4.94172 8.38095 4.57143 8.0381 4.57143 7.61905C4.57143 7.2 4.93334 6.85715 5.39125 6.85715ZM8.32305 6.85715C8.78095 6.85715 9.14285 7.2 9.14285 7.61905C9.14285 8.0381 8.78095 8.38095 8.32305 8.38095C7.8735 8.38095 7.504 8.0381 7.504 7.61905C7.504 7.2 7.86515 6.85715 8.32305 6.85715ZM12.1074 0C12.9935 0 13.7143 0.736 13.7143 1.648V16L12.0297 14.48L11.0811 13.584L10.0777 12.6316L10.4937 14.112H1.60686C0.72076 14.112 0 13.376 0 12.464V1.648C0 0.736 0.72076 0 1.60686 0H12.1066H12.1074ZM9.08265 10.448C10.8145 10.3924 11.4811 9.232 11.4811 9.232C11.4811 6.656 10.352 4.56762 10.352 4.56762C9.2244 3.70438 8.1501 3.728 8.1501 3.728L8.0404 3.856C9.37295 4.272 9.99165 4.87238 9.99165 4.87238C9.26405 4.46246 8.4622 4.2014 7.63275 4.10438C7.1066 4.04496 6.5752 4.05008 6.0503 4.11962C6.00305 4.11962 5.96345 4.128 5.91695 4.13562C5.64265 4.16 4.976 4.26362 4.13791 4.64C3.84838 4.77562 3.67543 4.87238 3.67543 4.87238C3.67543 4.87238 4.3261 4.24 5.7364 3.824L5.6579 3.728C5.6579 3.728 4.58438 3.70438 3.456 4.56838C3.456 4.56838 2.32762 6.656 2.32762 9.232C2.32762 9.232 2.98591 10.3916 4.71772 10.448C4.71772 10.448 5.00725 10.0884 5.24345 9.7844C4.24762 9.4796 3.872 8.8396 3.872 8.8396C3.872 8.8396 3.94972 8.896 4.09067 8.976C4.09829 8.9836 4.10591 8.992 4.12191 8.9996C4.14553 9.0164 4.16915 9.024 4.19276 9.04C4.38857 9.152 4.58438 9.2396 4.76419 9.312C5.0857 9.44 5.4697 9.568 5.91695 9.6564C6.5869 9.7877 7.27575 9.7903 7.94665 9.664C8.33745 9.59425 8.71875 9.4789 9.08265 9.3204C9.35695 9.216 9.6625 9.0636 9.98405 8.848C9.98405 8.848 9.5924 9.504 8.56535 9.8004C8.80075 10.1044 9.08345 10.448 9.08345 10.448H9.08265Z"
+      fill="white"
```

Comment:
Let's change the fill to `currentColor`, and set the color class on the icon.
```
<IconDiscordSocial class="text-white" />
```

Reply:
Sorry, forgot to modify them after exporting from figma, should be good now.